py/desispec/scripts/zproc.py

```diff
@@ -870,7 +886,7 @@ def main(args=None, comm=None):
             log.info(goodbye)
 
     if error_count > 0:
```

Comment:
since we're returning in both instances, we can remove the if statement and just `return int(error_count)`